types/fluxible/index.d.ts

```diff
@@ -90,7 +90,7 @@ export class Fluxible {
      * @param callback
      * @async Rehydration may require more asset loading or async IO calls
      */
-    rehydrate(state: any): void;
+    rehydrate(state: any, callback: (err: Error, context?: FluxibleContext) => void): void;
```

Comment:
Did you mean to make this optional?
http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks

Reply:
No, I meant to make the callback itself optional. I'll prepare another PR.

Reply:
You are right, I meant to make the **callback** itself optional. Updating it.

Reply:
Fixed, thanks a lot.